spacy/cli/info.py

```diff
@@ -35,11 +37,20 @@ def info(
     markdown: bool = False,
     silent: bool = True,
     exclude: Optional[List[str]] = None,
+    url: bool = False,
 ) -> Union[str, dict]:
     msg = Printer(no_print=silent, pretty=not silent)
     if not exclude:
         exclude = []
-    if model:
+    if url:
+        if model is not None:
+            title = f"Download info for pipeline '{model}'"
+            data = info_model_url(model)
+            print(data["download_url"])
```

Comment:
Shouldn't we print through `msg` here (and at L67)?

Reply:
I was originally using `msg`, but I changed it because I expect a use like this:

```bash
download_url=$(spacy info en_core_news_sm)
wget $download_url
```

So any extra formatting would be undesirable. 

I think L67 is that way for similar reasons, but I'm not sure.